usr/bin/tce-update

```diff
@@ -47,11 +49,7 @@ upgrade(){
 				echo -n " ${YELLOW}$TARGET${NORMAL}"
 			else
 				echo -n  "Updating $TARGET "
-				if [ -n "$USEZSYNC" ]; then
-					fetchzsync "$TARGET" 2>/dev/null
-				else
-					fetch "$TARGET" 2>/dev/null
-				fi
+				fetchzsync "$TARGET" 2>/dev/null
```

Comment:
Why move the check to see if we should use zsync inside the fetchzsync function.  I do not see the need to change these lines.  If its not using zsync we should not call fetchzsync.